wpeverest/everest-forms
# Enhancement - Captcha design Revamp.

### All Submissions:






### Changes proposed in this Pull Request:
This PR enhances the design of Captcha Settings.

### How to test the changes in this Pull Request:

1. Go to > Global Settings > Captcha
2. Check the captcha working properly or not.


### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Captcha design Revamp.
